### ISSUE TITLE
Unable to start HPX Windows due to Elastic Search

### ISSUE THREAD

**Issue by Dystasia** [happypandax/happypandax#209]
**Describe the issue**
Unable to start HPX server due to elastic search JVM not starting.

**To Reproduce**
Start Server

**Logs**
Aug-08 03:06:34--INFO   [server].happypanda.core.esearch: Starting builtin ESearch server... 
Aug-08 03:06:34--INFO   [server].happypanda.core.esearch: Waiting on ESearch for maximum 60 seconds... 
Aug-08 03:07:37--ERROR  happypanda.main: Failed to configure built-in ESearch server, process exit code: 1 also see logs\esearch_stdout.log 
Aug-08 03:07:37--WARNING happypanda.main: Could not establish connection to ESearch server, aborting. 
Aug-08 03:07:37--INFO   happypanda.main: HPX END 
Aug-08 03:07:37--INFO   happypanda.common.hlogger: Logger shutting down. 

esearch_stdout.log:
Exception in thread "main" java.lang.RuntimeException: starting java failed with [1]
output:
[0.002s][error][logging] Error opening log file 'logs/gc.log': No such file or directory
[0.002s][error][logging] Initialization of output 'file=logs/gc.log' using options 'filecount=32,filesize=64m' failed.
error:
Invalid -Xlog option '-Xlog:gc*,gc+age=trace,safepoint:file=logs/gc.log:utctime,pid,tags:filecount=32,filesize=64m', see error log for details.
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.
    at org.elasticsearch.tools.launchers.JvmErgonomics.flagsFinal(JvmErgonomics.java:126)
    at org.elasticsearch.tools.launchers.JvmErgonomics.finalJvmOptions(JvmErgonomics.java:88)
    at org.elasticsearch.tools.launchers.JvmErgonomics.choose(JvmErgonomics.java:59)
    at org.elasticsearch.tools.launchers.JvmOptionsParser.jvmOptions(JvmOptionsParser.java:137)
    at org.elasticsearch.tools.launchers.JvmOptionsParser.main(JvmOptionsParser.java:95)

**Environment:**
 - Windows 10
 - HPX Version 0.13

**Additional context**
Fresh Install

**Comment by waz1500** [comment]
Same issue with a fresh install using the Docker environment. 
```
HPX_DATA was defined, data will now be at: /data
Using sqlite database
Creating a new database...
Creating default superuser 'default'
'Failed to configure built-in ESearch server, process exit code: 1 also see /data/logs/esearch_stdout.log '

'Could not establish connection to ESearch server, aborting. '
Sentry is attempting to send 0 pending error messages

Waiting up to 2 seconds
Press Ctrl-C to quit
```

**Comment by Dystasia** [comment]
I was able to temporally fix it by adding the gc.log file missing to bin\win32\elasticsearch\logs

Hope that helps others.

**Comment by twiddli** [comment]
should be fixed in `v0.13.1`
